### ISSUE TITLE
[BUG] Multi window divider button not responsive

### ISSUE THREAD

**Issue by weebawop** [ananjaser1211/FloydQ_Reports#189]
**Describe the bug**
The button between windows in split screen mode does not respond when tapped

**To Reproduce**
Have two apps running in split screen mode and try tapping the button in the centre of the divider

**Expected behavior**
A pop up should appear giving options ie swap window positions, but nothing shows up

**Smartphone (please complete the following information):**
 - Device: Galaxy S7 Edge
 - Model: G935F

**Comment by ananjaser1211** [comment]
This bug is quite weird, i can confirm it does not work on my S7, and works on my S9
However it does not work on some users S9 and N9.  

i doubt it is related to some goodlock feature too. i will try and figure out what causes it. but i dont think logs will contain any information

**Comment by weebawop** [comment]
If it's of any help, the problem only arose on V4. Also the divider doesn't show up in the middle of the screen when I toggle it via a shortcut, it starts at the bottom, out of sight, another V4 issue

**Comment by ananjaser1211** [comment]
> This bug is quite weird, i can confirm it does not work on my S7, and works on my S9
> However it does not work on some users S9 and N9.  
> 
> i doubt it is related to some goodlock feature too. i will try and figure out what causes it. but i dont think logs will contain any information

Today I noticed the same bug on my N8 with Q. While still working on my s9

I have took a look at my apps list and I realized i have multistar installed on the s9. 

To double check. I asked my friend on s10 and another s9 and N9 to see if the same issue happens. And indeed no menu shows up when pressing the divider.

Seems that Samsung has moved this functionality in multistar. Not sure if it's 2.1 exclusive or not tho.

I'll be closing this as it's not something I can bring back.

I will link this issue in then "oneui missing features" list on xda
